javatests/jsinterop/generator/externs/simpleclass/simpleclass.js

```diff
@@ -69,11 +69,24 @@ SimpleClass.wait = function() {};
  */
 SimpleClass.staticReadonlyProperty
 
+/**
+ * @deprecated
+ * @const {string}
+ */
+SimpleClass.staticDeprecatedProperty
```

Comment:
I'd name that `deprecatedConstant` and add:

```js
/**
 * @deprecated
 * @type {string}
 */
SimpleClass.deprecatedStaticProperty;
```

Reply:
done